topics/genome-annotation/tutorials/bacterial-genome-annotation/tutorial.md

```diff
@@ -0,0 +1,592 @@
+   > >    - AF503772 corresponds to a *Enterococcus faecalis* plasmid
+   > >    - CP003584 corresponds to a *Enterococcus faecium* plasmid
+   > > 
+   > > 4. All plasmid sequences corresponding to *Staphylococcus aureus* plasmids are all on contig00019, making this contig likely a plasmid. In addition, this contig has a length of 30,347 bp, which is similar to the expected length of the plasmid for KUN1163 in [Table 1](https://journals.asm.org/doi/10.1128/mra.01212-19#tab1) in {% cite Hikichi_2019 %}
+   > {: .solution}
+   {: .question}
+
+- `plasmid.fasta`: A fasta file containing the best matching sequences from the query genome
+- `hit_in_genome.fasta`: A fasta file containing the best matching plasmid genes from the database
+
+## Integrons
+
+Integrons are genetic mechanisms that allow bacteria to adapt and evolve rapidly through the stockpiling and expression of new genes. An integron is minimally composed of:
+
+- a gene encoding for a site-specific recombinase (intI)
+- a proximal recombination site (attI), which is recognized by the integrase and at which gene cassettes may be inserted
+- a promoter (Pc) which directs transcription of cassette-encoded genes
+
+To detect integrons, we will use **IntegronFinder** ({% cite neron2022integronfinder %}). This tool:
+
+1. Annotates the CDS with **Prodigal**
+2. Detects independently:
+
+   1. integron integrase using the intersection of two HMM profiles: one specific of tyrosine-recombinase (PF00589) and one specific of the integron integrase, near the patch III domain of tyrosine recombinases
+   2. attC recombination sites with a covariance model (CM), which models the secondary structure in addition to the few conserved sequence positions.
+
+3. Integrates the results to distinguish 3 types of elements
+
+   - Complete integron: Integron with integron integrase nearby attC site(s)
+   - In0 element: Integron integrase only, without any attC site nearby
+   - CALIN element: Cluster of attC sites Lacking INtegrase nearby 
+
+> <hands-on-title> Integron identification</hands-on-title>
+>
+> 1. {% tool [IntegronFinder](toolshed.g2.bx.psu.edu/repos/iuc/integron_finder/integron_finder/2.0.2+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Replicon file"*: Contig file
+>    - *"Thorough local detection"*: `Yes`
+>    - *"Search also for promoter and attI sites?"*: `Yes`
+>    - *"Remove log file"*: `Yes`
+>
+{: .hands_on}
+
+**IntegronFinder** generates 2 outputs:
+
+- A summary with for each sequence in the input the number of identified CALIN elements, In0 elements, and complete integrons.
+
+   > <question-title></question-title>
+   >
+   > How many integron elements have been found?
+   >
+   > > <solution-title></solution-title>
+   > >
+   > > No integron elements have been found on any contig. It could be because the genome is too stable, or because the assembly is not good enough quality and some parts useful for the integron detection were removed.
```

Comment:
```suggestion
   > > No integron elements have been found on any contig. It could be because the genome is too stable, or because the assembly quality is not good enough and some parts useful for the integron detection were removed.
```